connectors/data.json

```diff
@@ -0,0 +1,90 @@
+[
+    {
+        "name": "activedirectory",
+        "label": "Active Directory",
+        "category": [
+            "Directory Service"
+        ],
+        "description": "Active Directory (AD) is a directory service that Microsoft developed for Windows domain networks. You can directly query AD to retrieve information about users, groups, and computers, in an organization, by using the Lightweight Directory Access Protocol (LDAP) to directly query the AD.",
+        "publisher": "Fortinet",
+        "operation_roles": [],
+        "configurations": [
```

Comment:
Please remove the connector configuration

Reply:
Done